client/state/followers/test/selectors.js

```diff
@@ -57,7 +49,49 @@ describe( 'selctors', () => {
 				},
 			};
 			const query = { siteId: 555 };
-			expect( getFollowersForQuery( state, query ) ).to.be.null;
+			expect( getFollowersForQuery( state, query ) ).toBeNull;
+		} );
+	} );
+
+	describe( '#getIsFetchingFollowersForQuery', () => {
```

Comment:
Do we also need a test to cover the case when a query request with that serialized query doesn't exist at all?

Reply:
fixed via 31043440f5